opal/core/search/tests/test_search_query.py

```diff
@@ -715,15 +719,15 @@ def test_get_patient_summaries(self):
         query = queries.DatabaseQuery(self.user, self.name_criteria)
         summaries = query.get_patient_summaries()
         expected = [{
-            'id': self.patient.id,
+            'id': self.patient.demographics_set.first().id,
```

Comment:
?We were implicitly assuming that b/c singletons `patient.id` == `patient.demographics.id` ?